Source/WebCore/css/horizontalFormControls.css

```diff
@@ -0,0 +1,4 @@
+/* Form controls don't go vertical. */
```

Comment:
Does this need a big copyright comment block at the top? (Seems most UA sheet files do, but some don't.)

Reply:
Added (though this file is only meant to be temporary)